corehq/tests/app_test_db_dependencies.yml

```diff
@@ -5,6 +5,19 @@
 #
 # This means that replacing INSTALLED_APPS with the list of declared dependencies plus the app
 # should still allow `./manage.py test app` to pass.
+
+# apps listed in the 'default' section will be included for all other apps
+default:
+  - django.contrib.auth
+  - django.contrib.contenttypes
+  - django.contrib.sessions
+  - two_factor
+  - django_otp.plugins.otp_static
+  - django_otp.plugins.otp_totp
+  - corehq.couchapps
+  - corehq.apps.tzmigration
+  - couchforms
+  - corehq.apps.users
```

Comment:
updated

On 25 April 2016 at 11:15, Cory Zue <notifications@github.com> wrote:

> In corehq/tests/app_test_db_dependencies.yml
> <https://github.com/dimagi/commcare-hq/pull/11380#discussion_r60883318>:
>
> > @@ -5,6 +5,19 @@
> >  #
> >  # This means that replacing INSTALLED_APPS with the list of declared dependencies plus the app
> >  # should still allow `./manage.py test app` to pass.
> > +
> > +# apps listed in the 'default' section will be included for all other apps
> > +default:
> > +  - django.contrib.auth
> > +  - django.contrib.contenttypes
> > +  - django.contrib.sessions
> > +  - two_factor
> > +  - django_otp.plugins.otp_static
> > +  - django_otp.plugins.otp_totp
> > +  - corehq.couchapps
> > +  - corehq.apps.tzmigration
> > +  - couchforms
> > +  - corehq.apps.users
>
> [image: :+1:] nice. I'm ok if we just take out couchapps for now.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/11380/files/92584349671cffe1ff2ccec841a3a702c40e4587#r60883318>
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa